python/lsst/cp/pipe/ptc.py

```diff
@@ -108,6 +110,12 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Maximum value to of mean signal (in ADU) to consider.",
         default=9e6,
     )
+    initialNonLinearityExclusionThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Initially exclude data points that are more than a factor of this from being linear from"
+        " the iterative fit.",
+        default=0.25,
```

Comment:
Why was the default value chosen? Trial and error with the data set that you used to test the code?

Reply:
The docstring for the function applying this config parameter explains it more fully, and I think that's the right place to have longer explanations. If we're guarding against negative we should also enforce the range zero to one, but I think if people are doing things that wrong we've got bigger problems 🙂

Reply:
Or rather, and less flippantly, config options are there to be changed by people who know what they're doing, and with good reason, and most places in the code don't bother checking too carefully whether someone's done something really stupid there.

Reply:
OK, fine, in writing that I convinced myself that's not ideal, so I'll change this to a `pexConfig.RangeField`!

Reply:
Oh, I forgot, I also update the docstring on the config param to read:
```        doc="Initially exclude data points with a variance that are more than a factor of this from being"
            " linear, from the PTC fit. Note that these points will also be excluded from the non-linearity"
            " fit. This is done before the iterative outlier rejection, to allow an accurate determination"
            " of the sigmas for said iterative fit.",```